libraries/botbuilder-ai/src/qnamaker-interfaces/qnamakerOptions.ts

```diff
@@ -83,4 +83,9 @@ export interface QnAMakerOptions {
      * includeUnstructuredSources - option to fetch answers from unsrtuctured sources
      */
     includeUnstructuredSources?: boolean;
+
+    /**
+     * Channel id for the user or bot on this channel
```

Comment:
Is this description correct? Channel Id?

Reply:
is the same description of the original variable